.github/workflows/e2e.yml

```diff
@@ -0,0 +1,44 @@
+name: E2E Tests
+on:
+  push:
+jobs:
+  e2e:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    strategy:
+      matrix:
+        e2e-config:
+          - standalone
+          # - turbo
```

Comment:
why doesn't turbo work?

Reply:
there's a type issue with react i believe

Reply:
dang

Reply:
that's cool that you can configure it separately from what you run locally